Samples/kotlin/build.gradle

```diff
@@ -21,6 +21,11 @@ android {
         }
     }
 
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
     packagingOptions {
         exclude 'META-INF/lib_release.kotlin_module'
```

Comment:
Can we remove this exclude pls? We don't need it anymore .

Reply:
Can do!